_data/meltano/utilities/dagster/quantile-development.yml

```diff
@@ -0,0 +1,51 @@
+name: dagster
+label: Dagster
+domain_url: https://dagster.io/
+namespace: dagster
+pip_url: dagster-ext
+ext_repo: https://github.com/quantile-development/dagster-ext
+maintenance_status: development
+repo: https://github.com/dagster-io/dagster
+executable: dagster_invoker
+logo_url: /assets/logos/utilities/dagster.png
+variant: quantile-development
+commands:
+  describe:
+    executable: dagster_extension
+    args: describe
+    description: Describe the Dagster Extension.
+  init:
+    args: initialize
+    executable: dagster_extension
+    description: Initialize the Dagster extension.
+  up:
+    args: -f $REPOSITORY_DIR/repository.py
+    executable: dagit_invoker
+    description: Start Dagster.
+settings:
+- name: dagster_home
+  label: Dagster Home
+  env: DAGSTER_HOME
+  value: $MELTANO_PROJECT_ROOT/.meltano
```

Comment:
@JulesHuisman there was also some discussions as part of the Airflow extension about where to put files and where to call "home". In [Airflow](https://github.com/meltano/hub/blob/33b8d89a3ec0868ff8b5a12a6411b1a9a5c2adbd/_data/meltano/utilities/airflow/apache.yml#L105) we ended up deciding to put files in `$MELTANO_PROJECT_ROOT/orchestrate/airflow` and use that directory as that AIRFLOW_HOME. 

I dont know if that translates exactly to Dagster and this also isnt blocking but wanted to call it out.

Reply:
Good call-out, @pnadolny13. Will there be any project artifacts installed to `DAGSTER_HOME` that users will want to commit back to the repo?

More info on what would be stored their would be helpful context.

Two more things:

- There's a new environment variable for the `.meltano` folder. I don't recall the new env var name, but it should be in docs. Also, if dagster would drop files directly to `$DAGSTER_HOME`, it might be preferrable and safer to use a subdirectory of `.meltano` - although admittedly this creates a question of which process would pre-create that subdirectory.
- Another path forward would be to set DAGSTER_HOME to `$MELTANO_PROJECT_ROOT/orchestrators/dagster` and then in the `dagster:init` command we can generate that folder, along with a custom `$MELTANO_PROJECT_ROOT/orchestrators/dagster/.gitignore` that is specific to the dagster file naming patterns for temp/internal files.


Wdyt?

Reply:
@aaronsteers @JulesHuisman from what I see when I test this locally, the current iteration is doing everything right in terms of where it's putting files. Everything that we'd wnat to version control appears to be in the `orchestrate` directory and everything else is in `.meltano`. I'll make a suggestion to add the `/dagster` directory but otherwise I think we're good.

Reply:
@aaronsteers The DAGSTER_HOME contains just temporary files that are needed by Dagit (The dagster UI), so these shouldn't be commited to the repo. 

@tayloramurphy I can put it in a deeper subdirectory, I will just need to check if Dagster automatically creates this subdirectory.

Reply:
> @aaronsteers The DAGSTER_HOME contains just temporary files that are needed by Dagit (The dagster UI), so these shouldn't be commited to the repo.

Thanks for confirming this. 👍

Reply:
The `DAGSTER_HOME` location couldn't be changed to `.meltano/dagster` as Dagster doesn't create the `dagster` subdir. Is it an issue that it is not inside a subdir?

Reply:
@JulesHuisman - It's not a huge issue, but yes, there is some risk that dagster files could accidentally overlap with other subfolder names - but more likely it would just make that `.meltano` base directory a bit 'messier' - depending upon how many files/folders are created at that direct HOME path.

Since this is on the EDK, one simple workaround to the issue requiring the home dir to already exist would be to create a pre-run hook that creates `DAGSTER_HOME` if it does not exist.

(PS - Great job in the demo today! We're all very excited about this.)

Reply:
I really like the solution of the pre-hook, didn't think about that. I added it to the extension and updated the PR.